docs/reverse_proxy.md

```diff
@@ -38,9 +38,14 @@ the reverse proxy and the homeserver.
 server {
     listen 443 ssl;
     listen [::]:443 ssl;
+
+    # For the federation port
+    listen 8448 ssl default_server;
+    listen [::]:8448 ssl default_server;
+
     server_name matrix.example.com;
 
-    location /_matrix {
+    location / {
```

Comment:
the idea here is that people might have things other than synapse on port 443, hence only forwarding `/_matrix`. Suggest you leave this in place: it'll work find for the federation port too.
```suggestion
    location /_matrix {
```

Reply:
Sure — specifying `server_name` does at least make that less likely, but if it's functionally equivalent looks good.